src/core/Wizard/Wizard.test.tsx

```diff
@@ -2,8 +2,9 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import { render } from '@testing-library/react';
+import { fireEvent, render } from '@testing-library/react';
 import React from 'react';
+import { act } from '@testing-library/react';
```

Comment:
I wanted you to add `act` to line 5 imports 😀

Reply:
Just noticed that import... I have merged them now